assets/agenda/components/AgendaCoverageExistsFilter.jsx

```diff
@@ -26,13 +26,13 @@ function AgendaCoverageExistsFilter ({toggleFilter, activeFilter}) {
             <button
                 key='coverage-planned'
                 className='dropdown-item'
-                onClick={() => toggleFilter(filter.field, 'planned')}
+                onClick={() => toggleFilter(filter.field, gettext('planned'))}
```

Comment:
In this context, `toggleFilter` expects the same string that the server uses to construct the query https://github.com/superdesk/newsroom-core/blob/develop/newsroom/agenda/agenda.py#L466

Therefor using `gettext` here will cause a bug.

Reply:
The text was not being translated into French, adding `gettext` is doing the job. I am not sure about its aftereffects.

Reply:
The server is expecting the value `planned` when constructing the Elasticsearch query.
Using `gettext` here will change this value to something else, if English has not been selected